doc/sphinx/source/community/broken_recipe_policy.rst

```diff
@@ -0,0 +1,17 @@
+Broken recipe policy
+====================
+
+Recipes might stop to work for different reasons. Among those are, for instance, withdrawal of datasets
+used by the recipe, continued development of the ESMValTool including new features or retiring old ones as well as
```

Comment:
```suggestion
used by the recipe, backward incompatible development of the ESMValTool including new features or retiring old ones as well as
```
I'd stress the backward incompatible development here, as most developments shouldn't affect old recipes and it might sound more negative? Like the rest though, including the 3 releases and keeping it in the documentation (potentially add a new heading for "legacy" recipes?).
Great Work Axel!

Reply:
changes in the API might render old recipes non functional though - esmvalcore's API and other dependencies' API's

Reply:
Changed as suggested. Thanks @bettina-gier.